packages/public-docsite-setup/src/types.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
legacy project not following legacy rules - disabled for whole file instead inline disable for N interfaces